source/include/CEntryPoints.h

```diff
@@ -76,6 +76,7 @@ VIREO_EXPORT Boolean TypeRef_HasCustomDefault(TypeRef typeRef);
 VIREO_EXPORT EncodingEnum TypeRef_BitEncoding(TypeRef typeRef);
 VIREO_EXPORT Int32 TypeRef_Alignment(TypeRef typeRef);
 VIREO_EXPORT const char* TypeRef_Name(TypeManagerRef typeManager, TypeRef typeRef);
+VIREO_EXPORT const char* TypeRef_ElementName(TypeManagerRef typeManager, TypeRef typeRef);
```

Comment:
I wonder if we should change the signature of these to return an `EggShellResult` and stop using `cwrap`ing them.

Reply:
We can avoid cwrapping them after we enable reading SubString support so we aren't quite there yet